CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.13.0 Release
+- Release Date: October 16, 2018
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported
+
+**Combatibility Note: Mobile App v1.13+ is required for Mattermost Server v5.4+**
+
+### Highlights
+
+#### View Emoji Reactions
+- Hold down on any emoji reaction to see who reacted to the post.
+
+#### Hashtags
+- Added support for searching for hashtags in posts.
+
+#### Dropdown menus
+- Added support for dropdown menus in message attachments.
+
+### Improvements
+- Added support for iPhone XR, XS and XS Max.
+- Added support for nicknames on user profile.
+- On servers 5.4+, added support for searching in direct and group message channels using the "in:" modifier.
+- Channel autocomplete now gets closed if multiple tildes are typed.
+- Added a draft icon in sidebar and channel switcher for channels with unsent messages.
+- Users are now redirected to the archived channel view (rather than to Town Square) when a channel is archived.
+- When closing an archived channel, users are now returned to the previously viewed channel.
+
+### Bug Fixes
+- Refactored postlist to include Android Pie fixes and smoother scrolling.
+- Fixed an issue where deactivated users were not marked as such in "Jump To" search.
+- Fixed an issue where users got a permission error when trying to open a file from within the image preview screen.
+- Fixed an issue where session expiry notifications were not being sent on Android.
+- Fixed an issue where post attachments failed to upload.
```

Comment:
Should this say "- On servers 5.4+, fixed an issue where post attachments failed to upload."